packages/eslint/src/generators/utils/flat-config/ast-utils.spec.ts

```diff
@@ -124,6 +124,10 @@ describe('ast-utils', () => {
             ]
         }).map(config => ({
             ...config,
+            files: [
```

Comment:
This doesn't seem right? This will be setting files to _only_ resolve `cts` and `mts`? Unless eslint is doing some magical merging behind the scenes?

Reply:
This one specifically is when converting eslintrc to flat config when called from `convertEslintJsonToFlatConfig`. In those cases the `.ts` and `.tsx` extensions are already specified. I could add it to the ast utils as well just in case.

Reply:
I updated it.